trieste/acquisition/function.py

```diff
@@ -839,13 +848,22 @@ def get_lipschitz_estimate(
             self._lipschitz_constant = lipschitz_constant
             self._eta = eta
```

Comment:
what's eta in the non-EI case?

Reply:
eta is needed for the penalizer, regardless of the base acq